system/bootstrap.php

```diff
@@ -106,18 +107,9 @@
 
 require_once SYSTEMPATH . 'Autoloader/Autoloader.php';
 require_once SYSTEMPATH . 'Config/BaseService.php';
-require_once SYSTEMPATH . 'Config/Services.php';
-require_once APPPATH . 'Config/Services.php';
 
-// Use Config\Services as CodeIgniter\Services
-if (! class_exists('CodeIgniter\Services', false))
-{
-	class_alias('Config\Services', 'CodeIgniter\Services');
-}
-
-$loader = Services::autoloader();
-$loader->initialize(new Autoload(), new Modules());
-$loader->register(); // Register the loader with the SPL autoloader stack.
+// Initialize and register the loader with the SPL autoloader stack.
+BaseService::autoloader()->initialize(new Autoload(), new Modules())->register();
```

Comment:
See comment above for `BaseService`.

I don't know for the class aliasing if it is ever used in the wild but it should stay unless there is a negligible chance that users are using the aliased version.